### ISSUE TITLE
Extra spacing added in sub lists when separating lists with a new-line

### ISSUE THREAD

**Issue by kyldvs** [notable/notable#950]
- **OS Version**: Windows (Version 1809)
- **Notable Version**: 1.8.2

### Current behavior

When rendering this, there is normal spacing between the main bullets and the sublist (Foo, Bar):

```
# Test

- List 1 - Part 1
  - Foo
  - Bar
- List 1 - Part 2
  - Foo
  - Bar
- List 1 - Part 3
```

![Screenshot (23)](https://user-images.githubusercontent.com/1830497/70955702-a5c1a980-2026-11ea-83a0-df58d37183b7.png)

---

Compared to this, when trying to make two different lists there is extra spacing added between "List 1- Part 1" and the sub-item "Foo"

```
# Test

- List 1 - Part 1
  - Foo
  - Bar

- List 2 - Part 1
  - Foo
  - Bar
- List 2 - Part 2
```

![Screenshot (24)](https://user-images.githubusercontent.com/1830497/70955774-d6094800-2026-11ea-8d20-3cd51da07ac0.png)

### Expected behavior

There should not be extra spacing between "List 1- Part 1" and the sub-item "Foo", and other parts of the lists.

**Comment by fabiospampinato** [comment]
Upstream issue: https://github.com/commonmark/commonmark-spec/issues/628

**Comment by fabiospampinato** [comment]
And that, like pretty much everything else in the CommonMark spec, seems to be very difficult to get changed.

In a few years time, if things go very well, I'd like to explore the possibility of creating a more modern Markdown specification, there are so many things in CommonMark that don't make a lot of sense to me.

**Comment by kyldvs** [comment]
Yeah a more modern spec where we can actually fix weird behaviors like this might be better 😛

**Comment by fabiospampinato** [comment]
> (feel free to close this btw if it's not getting fixed upstream, I don't know if you like long-open issues)

I'd still consider this a bug, so let's keep it open.
